packages/components/src/design-language/typography.ts

```diff
@@ -0,0 +1,19 @@
+export const fontSizes = [
```

Comment:
Should we import this from `common/design-language` instead of copying it here?

Reply:
Makes sense! Done :)